pkg/sdk/datatypes/data_types_test.go

```diff
@@ -1095,6 +1097,88 @@ func Test_ParseDataType_Vector(t *testing.T) {
 	}
 }
 
+func Test_ParseDataType_Table(t *testing.T) {
+	type column struct {
+		name       string
+		legacyType string
+	}
+	type test struct {
+		input           string
+		expectedColumns []column
+	}
+
+	positiveTestCases := []test{
+		{input: "TABLE()", expectedColumns: []column{}},
+		{input: "TABLE ()", expectedColumns: []column{}},
+		{input: "TABLE(arg_name NUMBER)", expectedColumns: []column{{"arg_name", NumberLegacyDataType}}},
```

Comment:
please add `TABLE(arg_name NUMBER(38))` (i.e. single argument - it should work)

also: `TABLE(arg_name NUMBER(38), arg_name_2 VARCHAR)`